<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportFileNameTemplateWidget</class>
 <widget class="QWidget" name="ExportFileNameTemplateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Name Template</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Template: </string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_filename</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_filename"/>
       </item>
       <item>
        <widget class="QLabel" name="label_file_extension">
         <property name="text">
          <string>.placeholder</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>A filename template uses printf-like format strings to specify where the changing per-frame values should appear in the filename.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget_template">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="indentation">
      <number>10</number>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Format</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>%n</string>
      </property>
      <property name="text">
       <string>frame number, in the range [1, N]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>%u</string>
      </property>
      <property name="text">
       <string>frame index, in the range [0, (N-1)]</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>%f</string>
      </property>
      <property name="text">
       <string>reconstruction time of frame</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>%d</string>
      </property>
      <property name="text">
       <string>reconstruction time of frame (rounded to integer)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>%A</string>
      </property>
      <property name="text">
       <string>anchor plate id</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>%R</string>
      </property>
      <property name="text">
       <string>default reconstruction tree layer name</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>%D</string>
      </property>
      <property name="text">
       <string>date format &quot;yyyy-MM-dd&quot;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>%T</string>
      </property>
      <property name="text">
       <string>time format &quot;hh-mm-ss&quot;</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>The precision of %f and padding of %d may be customised using printf conversion specifiers (for example, &quot;%0.2f&quot; or &quot;%02d&quot;). %n and %u will be padded automatically to the width of the highest value.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
